src/ed_object_models/conversion_sdf.py

```diff
@@ -26,6 +28,15 @@ class bcolors:
     UNDERLINE = "\033[4m"
 
 
+@contextmanager
+def last_model_file(model_file: str):
+    global LAST_MODEL_FILE
+    old_last_model_file = LAST_MODEL_FILE
+    LAST_MODEL_FILE = model_file
+    yield
+    LAST_MODEL_FILE = old_last_model_file
+
+
```

Comment:
What is the purpose of this?

Reply:
As models can be defined in their own file, but also in a shared file. Therefore a model path doesn't always exist for a model. So when resolving "$(file FILENAME)", you need the path of the file it is in. As the the model path isn't available, we need to track the last opened file.